main.py

```diff
@@ -0,0 +1,95 @@
+from typing import Annotated
+from fastapi import Depends, FastAPI, Form
+import uvicorn
+from fastapi.middleware.cors import CORSMiddleware
+from pydantic import BaseModel
+import pymysql
+import bcrypt                                       # the library that we are going to use for hashing passwords
+import os
+from dotenv import load_dotenv, dotenv_values
+load_dotenv()
+
+rds_instance_pswd = os.getenv('RDS_INSTANCE_PSWD')
+
+"""
+Steps for storing env variable for RDS_Instance:
+1) create .env file in this python file's directory
+2) enter RDS_INSTANCE_PSWD = 'your_local_rds_instance_password'
+3) Don't forget to include .env file in .gitignore file 
+"""
+
+
+def get_db_connection(db):
+    connection = pymysql.connect(
+        host='localhost',
+        user='root',
+        password=rds_instance_pswd,
+        database=db,
+        cursorclass=pymysql.cursors.DictCursor
+    )
+    return connection
+
+app = FastAPI()
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=['*']
+)
+
+@app.post("/authorize/")
+async def login(email: Annotated[str, Form()], password: Annotated[str, Form()], utype: Annotated[str, Form()]):
+    con = get_db_connection(utype.lower())
+    table = 'org_tab' if utype.lower() == 'organiser' else 'user_tab'
+    cur = con.cursor()
+    sql = f"select name, hashedpswd from {table} where email = '{email}'"
+    cur.execute(sql)
+    result = cur.fetchone()
+    con.close()
+    if result is not None:
+        if bcrypt.checkpw(password.encode('utf-8'), result['hashedpswd'].encode('utf-8')):
+            return {"message": f"Authorization Successful!", "User": result['name']}
```

Comment:
Can we try to add a status code as part of the response as well?
200 - Success
401 - Unauthorized
404 - User not found